git-info-script.mjs

```diff
@@ -0,0 +1,10 @@
+import  shell from 'shelljs';
```

Comment:
Change the name of this file to `git-info-generator.mjs` so intention is clearer